sdks/java/extensions/sql/jdbc/build.gradle

```diff
@@ -34,12 +34,13 @@ dependencies {
   compile project(path: ":beam-sdks-java-extensions-sql", configuration: "shadow")
   compile "jline:jline:2.14.6"
   compile "sqlline:sqlline:1.4.0"
-  compile library.java.slf4j_jdk14
   compile library.java.guava
   testCompile project(path: ":beam-sdks-java-io-google-cloud-platform", configuration: "shadowTest")
   testCompile library.java.junit
   testCompile library.java.hamcrest_core
   testCompile library.java.hamcrest_library
+  shadowTestRuntimeClasspath library.java.slf4j_jdk14
```

Comment:
I don't believe this module is ever meant to be used as a library that is consumed by others and this is why the compile scope is used for slf4j_jdk14

Reply:
done

Reply:
@lukecwik Thanks for catching these.